package.json

```diff
@@ -81,5 +81,61 @@
     "@opentelemetry/tracing": "^0.18.0",
     "jaeger-client": "^3.15.0",
     "semver": "^7.1.3"
+  },
+  "peerDependencies": {
+    "@opentelemetry/instrumentation-dns": "^0.14.0",
+    "@opentelemetry/instrumentation-express": "^0.14.0",
+    "@opentelemetry/instrumentation-graphql": "^0.14.0",
+    "@opentelemetry/instrumentation-grpc": "^0.18.0",
+    "@opentelemetry/instrumentation-hapi": "^0.14.0",
+    "@opentelemetry/instrumentation-http": "^0.18.0",
+    "@opentelemetry/instrumentation-ioredis": "^0.14.0",
+    "@opentelemetry/instrumentation-koa": "^0.14.0",
+    "@opentelemetry/instrumentation-mongodb": "^0.14.0",
+    "@opentelemetry/instrumentation-mysql": "^0.14.0",
+    "@opentelemetry/instrumentation-net": "^0.14.0",
```

Comment:
Btw I think this is not published, or is it? At least I wasn't able to find it on npm

Reply:
It is but we were not including it before so it won't be on npm yet. The next release should have it.

Reply:
It is not. I think mysql is not either but including them is harmless as we don't try to auto install these. Things should work for users once these are published and users install them.

Reply:
Or do you prefer we remove these for now and issue a patch release later once they are published?